coffee_cmd/src/cmd.rs

```diff
@@ -14,6 +14,8 @@ pub struct CoffeeArgs {
     pub network: Option<String>,
     #[clap(short, long, value_parser, name = "data-dir")]
     pub data_dir: Option<String>,
+    #[clap(short, long, action = clap::ArgAction::SetTrue)]
+    pub skip_verify: bool,
```

Comment:
the default is always fails for this right?

Reply:
yes the default for `skip_verify` is false (we do verify that coffee is sane)